src/test/java/AppHealthCheckTest.java

```diff
@@ -0,0 +1,23 @@
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AppHealthCheckTest {
+
+    @Test
+    public void isAppAlive() throws Exception {
```

Comment:
How does this unit test can help us?

Reply:
The test quickly validates that the application container is running and the web server is reachable, so we know we can start coding against it. While the test is simple, it can catch glaring issues like if the server isn't running or is misconfigured, which saves debugging time later. Although this is a basic test, it serves as a foundation for more complex health checks in the future.